src/coupling/assembly_base.hh

```diff
@@ -40,13 +40,18 @@ public:
     typedef typename GenericAssemblyBase::CouplingIntegralData CouplingIntegralData;
     typedef typename GenericAssemblyBase::BoundaryIntegralData BoundaryIntegralData;
 
-	/// Constructor
-	AssemblyBase(unsigned int quad_order) {
+    /// Constructor
+    AssemblyBase(unsigned int quad_order)
+    : min_edge_sides_(2),
```

Comment:
Delegate array initialization to AssemblyBase(). 
Try to add a virtual method to get the initial configuration of sizes.
Default implementation in AssemblyBase.
Add there comment where these numbers come from.